ps_imageslider.php

```diff
@@ -188,6 +189,7 @@ protected function createTables()
               `id_homeslider_slides` int(10) unsigned NOT NULL AUTO_INCREMENT,
               `position` int(10) unsigned NOT NULL DEFAULT \'0\',
               `active` tinyint(1) unsigned NOT NULL DEFAULT \'0\',
+              `expiration` datetime NOT NULL,
```

Comment:
I think we shouldn't force an expiration date, some of the slides may stay here indefinitely